books/free-programming-books-langs.md

```diff
@@ -2000,6 +2000,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Django Official Documentation](https://buildmedia.readthedocs.org/media/pdf/django/2.2.x/django.pdf) (PDF) (2.2)
 * [Django Official Documentation](https://buildmedia.readthedocs.org/media/pdf/django/3.1.x/django.pdf) (PDF) (3.1)
 * [Django Official Documentation](https://buildmedia.readthedocs.org/media/pdf/django/3.2.x/django.pdf) (PDF) (3.2)
+* [Django Official Documentation](https://buildmedia.readthedocs.org/media/pdf/django/4.1.x/django.pdf) (PDF) (4.1)
```

Comment:
Duplicate of #8464. Do we need to have official documentation of each version?

Reply:
The list contains documentation for django 3.1 and 3.2 so included minor releases 4.0 and 4.1

Reply:
I think only major releases at this point